### ISSUE TITLE
Regenerate cached attachments so that they get <title> tags

### ISSUE THREAD

**Issue by kingqueen3065** [mysociety/alaveteli#4071]
Google Search Console identifies that there are no HTML title tags for HTML versions of PDF, TXT etc. documents, e.g. https://www.whatdotheyknow.com/request/154598/response/381797/attach/html/2/Employee%20handbook%20Feb%20131.pdf.html

Adding a title tag would likely improve Google search results for said material. Perhaps have the title of the request to which it is attached? or the filename or something?

**Comment by garethrees** [comment]
They [do](https://www.whatdotheyknow.com/request/401913/response/983427/attach/html/4/REQ02625.pdf.html), as of around https://github.com/mysociety/alaveteli/commit/ea1e040780f00938331e92472780c91b7e0f43a2#diff-dc48ad4750a95626efc6b2a6655af4c6R4. That html page is cached from some point before the `<title>` tags got introduced.

We could blow away the caches of old attachments – I don't think we'd need to be too worried about letting fresh caches get generated on demand. Do we get a list of URLs that the search console things are a problem?

Incidentally I've just noticed the old logo, so opened https://github.com/mysociety/whatdotheyknow-theme/issues/410.

**Comment by kingqueen3065** [comment]
yup - spreadsheet https://docs.google.com/spreadsheets/d/1RxyoH_A7axmvay0onKOKEgYZYKNDoYUq2h6IUbjkp_g/edit?usp=sharing @garethrees

**Comment by garethrees** [comment]
We've now started clearing attachment caches automatically after a while, so this effectively isn't a problem.

We've done this through a cron job which just removes any files created after N days, so we haven't fixed the problem within Alaveteli itself. #1006 is where any further improvements should take place.

I loaded a few attachments from the spreadsheet above and all were regenerated with a `<title>` tag (except actual `.html` attachments – always an edge case!)
